run.py

```diff
@@ -1,21 +1,83 @@
-# coding=utf-8
+#!/usr/bin/env python3.6
+
+
+# stdlib
 import os
 import sys
-# from werkzeug.debug import DebuggedApplication
+import argparse
+import signal
+import time
+
+# 3rd
+from gevent.wsgi import WSGIServer
+import livereload
+
+# local
+from application import app, configure
+
+
+PORT = os.environ.get('PORT') or '9999'
+HOST = os.environ.get('HOST') or 'localhost'
+
+
+def main(args):
+    configure(os.getenv('FLASK_CONFIG') or 'default')
+
+    if args.test is True:
+        pid = os.fork()
+        if pid == 0:
+            run_server(mode='debug')
+        else:
+            os.system(f'py.test')
```

Comment:
Is there a way to run this directly without making a system call?

Reply:
Erhm, subprocess is another option. Preferences?

I want to use shell though as we need the test script to be compatible on any device/platform.

Reply:
https://docs.pytest.org/en/latest/usage.html#calling-pytest-from-python-code

Reply:
*nifty!* I'll implement.